src/extension/src/controller.ts

```diff
@@ -190,21 +194,30 @@ export class Controller {
     });
   }
 
+  private static getDefaultOutputPath() {
+    const userOS = os.platform();
+    const command: string = this.isUnix(userOS) ? "~" : "%USERPROFILE%";
+    const outputPath = child_process
+      .execSync("echo " + command)
+      .toString()
+      .trim();
+
+    Controller.reactPanelContext.postMessageWebview({
+      command: ExtensionCommand.GetOutputPath,
+      payload: {
+        outputPath: outputPath
+      }
+    });
+  }
+
+  private static isUnix(userOS: string): boolean {
+    return userOS === "linux" || userOS === "darwin";
+  }
+
   private static loadUserSettings() {
-    let outputPathDefault = vscode.workspace
```

Comment:
Would it still be valuable to be able to override the default directory on the extension settings?

Reply:
Good questions, I'll talk to the team about it